src/components/Header.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+export default function Header() {
+  const [currentPage, setCurrentPage] = useState(1);
+  return (
+    <header className="container p-3 border-bottom shadow fixed-top text-bg-success mt-2 rounded">
+      <nav className="d-flex align-items-center justify-content-between">
+        <div className="d-flex gap-2 align-items-center">
+          <img id="logo" src="./planet.png" alt="Planet Logo" />
+          <h2>Space Travelers Hub</h2>
+        </div>
+        <div className="btn-group">
+          <Link
+            to="/"
+            className={`btn btn-outline-light ${
+              currentPage === 1 && 'active'
+            }`}
+            onClick={() => setCurrentPage(1)}
+          >
+            Rockets
+          </Link>
+
+          <Link
+            to="/missions"
+            className={`btn btn-outline-light ${
+              currentPage === 2 && 'active'
+            }`}
+            onClick={() => setCurrentPage(2)}
+          >
+            Missions
+          </Link>
+
+          <Link
+            to="/profile"
+            className={`btn btn-outline-light ${
+              currentPage === 3 && 'active'
+            }`}
```

Comment:
- [ ] Great job implementing the header :clap: However it would be nice to use `<NavLink />` for the page navigation. That will help easily style the active link using the its CSS `active` class. And it is also a requirement of the project (see image)
![Screenshot from 2023-07-26 03-04-48](https://github.com/M-Anwar-Hussaini/Space-Travelers-Hub/assets/91142432/f7c44e03-7204-4206-8342-8559538c042c)

  Great work creating the `setCurrentPage` function but it has a bug. As you can see on this image :point_down: the user is on the profile page but the `Rockets` link is the one that has been selected. Therefore I suggest you use `<NavLink />` instead of `<Link />`
![Screenshot from 2023-07-26 03-11-52](https://github.com/M-Anwar-Hussaini/Space-Travelers-Hub/assets/91142432/3ff1264f-a563-4111-ac10-4327958669b8)

Reply:
Hi @LionRouge1. 
Thanks a lot for reviewing our project. We are very grateful to you for providing your constructive feedback and we'll apply the changes needed.